.azure-pipelines/nightly.yml

```diff
@@ -21,6 +21,8 @@ extends:
   parameters:
     pool:
       name: MSEngSS-MicroBuild2022-1ES
```

Comment:
The MicroBuild team told me the pool `MSEngSS-MicroBuild2022-1ES` is deprecated, and suggest us to use the new pool `VSEngSS-MicroBuild2022-1ES`.